gallery/settings.py

```diff
@@ -78,7 +78,7 @@
     'default': {
        'ENGINE': 'django.db.backends.postgresql',
        'NAME': 'galleriesdb',
-       'USER': 'postgres',
+       'USER': os.getenv('galleryUser'),
```

Comment:
no check if os.getenv return None, should be wrapped in some function which returns empty string instead or handled gracefuly